src/Symfony/Component/Pgp/Exception/PgpBadPassphraseException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Pgp\Exception;
+
+/**
+ * @author PuLLi <the@pulli.dev>
+ */
+class PgpBadPassphraseException extends InvalidArgumentException
```

Comment:
Not sure we need the `PgP` prefix, as we have the namespace of the component

Reply:
That's a good point!

Reply:
The parent `Symfony\Component\Pgp\Exception\RuntimeException` is not useful. You can remove it.
```suggestion
class BadPassphraseException extends \InvalidArgumentException implements ExceptionInterface
```